### PR TITLE
Request cherry pick approver for Console kubevirt plugin

### PR DESCRIPTION
**Pull request by gouyang**
Hi,
I'm CNV qe, backport is hard for console kubevirt plugin as we need to ping OCP guys to label cherry-pick-approved, may I join the list?

Thanks,
Gouyang

### CODE REVIEW

**Comment by gouyang** [comment]
cc:// @spadgett @yaacov @tnisan

**Comment by yaacov** [comment]
LGTM

**Comment by yaacov** [comment]
@yapei PTAL

**Comment by yapei** [comment]
LGTM

**Comment by yaacov** [comment]
@spadgett hi, do you know  who should review QE cherry pick approver requests ?

**Comment by spadgett** [comment]
I can approve, but the PR has a merge conflict.

**Comment by gouyang** [comment]
@spadgett Rebased, could you approve this?

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by gouyang** [comment]
Thanks, @spadgett

**Pull request merged**
